tests/test_django.py

```diff
@@ -932,39 +931,35 @@ def generate(cls):
 
 class PreventChainedSignalsTestCase(django_test.TestCase):
 
-    def test_class_decorator_with_muted_subfactory(self):
-        @receiver(signals.post_save, sender=models.PointedModel)
-        def boom(instance, created, raw, **kwargs):
-            raise Exception("BOOM!")
+    def setUp(self):
+        self.post_save_mock = mock.MagicMock(side_effect=Exception('BOOM!'))
```

Comment:
I have a couple of questions here:
 * Is the side_effect required if you have explicit asserts for not called?
 * Would it work with a classic Mock instead of a MagicMock?

Reply:
Hey @federicobond, thanks for the review!

`Is the side_effect required if you have explicit asserts for not called?`:
It isn't required, I mainly added it as a way of ensuring test failure in the event of the assert being removed. I am open to removing the side effect, I do this kind of thing to defend against testing refactors.

`Would it work with a classic Mock instead of a MagicMock?`
Just changed it to `Mock` and everything is good. I will push that update now.

Reply:
Thanks for the prompt response. I am not too worried about defending against test refactors, but I don't mind it either.